website/src/css/customTheme.css

```diff
@@ -23,7 +23,8 @@
   --ifm-font-family-monospace: "Space Mono", monospace;
   --ifm-blockquote-color: var(--text-secondary);
 
-  --ifm-container-width-xl: 1022px;
+  --ifm-container-width: 2000px;
+  --ifm-container-width-xl: 2000px;
```

Comment:
Modifying this will also change the container width for the docs, can we add some class like "blog-wrapper" to specify the width for blogs only?

Reply:
- https://github.com/nervosnetwork/docs.nervos.org/issues/375